sys/cddl/dev/dtrace/dtrace_ioctl.c

```diff
@@ -40,6 +54,14 @@ dtrace_ioctl_helper(struct cdev *dev, u_long cmd, caddr_t addr, int flags,
 	dhp = NULL;
 	dof = NULL;
 	rval = 0;
+	printf("%s - not implemented\n", __func__);
+	return ENOSYS;
+	/*
+	 * According to sys/cddl/contrib/opensolaris/uts/common/sys/dtrace.h:
+	 * The DTRACEHIOC_ADD and DTRACEHIOC_REMOVE are left in place for
```

Comment:
This doesn't include DTRACEHIOC_ADDDOF which is still widely used (dtri.c and dt_pid.c), and DTRACEHIOC_REMOVE is still used by dtri.c. From reading this, my assumption is that DTRACEHIOC_ADDDOF is the new way to do DTRACEHIOC_ADD, but I don't know why there's only a deprecated DTRACEHIOC_REMOVE. Do you know about how this part of things works? My DTrace knowledge is rather limited.

Reply:
ops, I must have confused `ADD` with `ADDOF`. Will fix this